lib/lndclient/LndClient.ts

```diff
@@ -72,6 +73,25 @@ class LndClient extends BaseClient {
     });
   }
 
+  private connect = (lndCLient: LndClient, subscribeInvoices: boolean = false) => {
```

Comment:
You don't need the `lndClient` parameter here, you can just use `this`.

Reply:
after arrow func, that's true :)